Rakefile

```diff
@@ -1,63 +1,72 @@
-require 'rubygems'
-require 'bundler/setup'
-
 require 'puppetlabs_spec_helper/rake_tasks'
```

Comment:
same here, completely managed by modulesync:
https://github.com/voxpupuli/modulesync_config/blob/master/config_defaults.yml#L62-L71

let me know if any of the changes are needed and we will fiddle something out.